Dockerfile

```diff
@@ -2,7 +2,7 @@
 
 # Build stage: Install yarn dependencies
 # ===
-FROM node:18 AS yarn-dependencies
+FROM node:21 AS yarn-dependencies
```

Comment:
We used to only do LTS versions of node, which would be `20` in this case.

Reply:
ahh okay sure, I just copied this update from the renovate PR. Will update this to 20

Reply:
There is no easy way to tell renovate to only do node LTS "equal" numbers. I think at some point we had it hardcoded in renovate config for couple versions forward, but seems it's gone from the shared config now.

Reply:
I still have it in Vanilla: https://github.com/canonical/vanilla-framework/blob/main/renovate.json#L17-L21